content/200-concepts/100-components/02-prisma-client/053-client-extensions/120-query.mdx

```diff
@@ -129,6 +136,25 @@ const xprisma = prisma.$extends({
 })
 ```
 
+### Modify all Prisma Client operations
+
+Use the `$allOperations` method to modify all query methods present in Prisma Client. The `$allOperations` can be used on both model operations and raw queries.
+
+You can modify all methods as follows:
+
+```ts
+const xprisma = prisma.$extends({
+  query: {
+    $allOperations({ model, operation, args, query }) {
```

Comment:
OT: @SevInf @millsp Why the mismatch `query` and `operations` here btw? Do we call the same thing two different names on different levels?

Reply:
`query` and `operation` are two different things:
- `operation` - name of the operation performed, i.e.  `findFirst`
- `query` is a function for performing the actual query, (`query(args)`)

And they exist in every query callback.

Reply:
Isn't the `query` a `findFirst` in this example?